src/toast/toast.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import { onlyUpdateForPropTypes } from 'recompose'
+import styles from '../../src/toast/toast.styles.scss'
+
+const Toast = ({ style, className, title, button, print, close, delay, position, type }) => {
+  let timeout = null
+  const classes = classnames(styles.toast, className, {
+    [styles.top]: position === 'top',
+    [styles.bottom]: position === 'bottom',
+    [styles.print]: print,
+    [styles.success]: type === 'success',
+    [styles.warning]: type === 'warning',
+    [styles.error]: type === 'error',
+  })
+
+  if (print) timeout = setTimeout(close, delay)
```

Comment:
What if an other prop is modified in the mean time ?
Like the className for animation ?

You will trigger multiple `close` and then multiple `remove`

Reply:
I'm thinking of doing an HOC to manage this behavior.